microsoft/testsuites/network/stress.py

```diff
@@ -100,6 +103,27 @@ def verify_stress_sriov_iperf(self, environment: Environment) -> None:
         # 3. Do VF connection test.
         sriov_vf_connection_test(environment, vm_nics)
 
+    @TestCaseMetadata(
+        description="""
+        This case verify VM works well after disable and enable accelerated network in
+         network interface through sdk under stress.
+
+        Steps,
+        1. Do the basic sriov check.
+        2. Set enable_accelerated_networking as False to disable sriov.
+        3. Set enable_accelerated_networking as True to enable sriov.
+        4. Do the basic sriov check.
+        """,
+        priority=3,
+        timeout=7200,
+        requirement=simple_requirement(
+            network_interface=features.Sriov(),
+            supported_platform_type=[AZURE, READY],
```

Comment:
Remove Ready, because this test case used the feature, so it cannot run on Ready environment.